pkg/server/image_remove.go

```diff
@@ -53,26 +53,39 @@ func (c *criContainerdService) RemoveImage(ctx context.Context, r *runtime.Remov
 		}
 		desc, err := cImage.Config(ctx)
 		if err != nil {
-			return nil, fmt.Errorf("failed to get image %q config descriptor: %v", tag, err)
+			// We can only get image id by reading Config from content.
+			// If the config is missing, we will fail to get image id,
+			// So we will can't remove the image forever,
```

Comment:
won't be able to
I'll fix in another PR.

Reply:
It will waste your many time.
You may wait for my modification. just for a while.
I will be responsible in the end. :)

Reply:
Sure. Will wait for your PR then.